app/main.py

```diff
@@ -323,29 +356,12 @@ def clean_login_background(is_modal_opened):
         raise PreventUpdate
 
     else:
-        return ''
+        return '', {}
 
 
 # ----------------------------------------------------------------------------------------------------------------------
 # Callbacks related to the "Alerts and Infrastructure" view
 
-@app.callback(
-    Output('alerts_info', 'children'),
-    Input('geojson_departments', 'hover_feature')
-)
-def hover_department_alerts(hovered_department):
-    """
-    -- Displaying department name in the alerts view --
-
-    This callback detects what department is being hovered by the user's cursor.
-
-    It returns the corresponding name in the info object in the upper right corner of the map.
-    """
-    if hovered_department is not None:
-        return build_info_box(hovered_department)
-    else:
-        return build_info_box()
-
```

Comment:
😢

Reply:
I know .... This could be reverted if necessary in the future 😅